pantry/src/server.rs

```diff
@@ -72,7 +72,7 @@ struct JobPollResponse {
 /// Poll to see if a Pantry background job is done
 #[endpoint {
     method = GET,
-    path = "/crucible/pantry/0/job/{id}/is_finished",
+    path = "/crucible/pantry/0/job/{id}/is-finished",
```

Comment:
Same note here, I think Omicron uses the generated client and will be updated wholesale at the same time the pantry server is.